src/const.ts

```diff
@@ -11,3 +11,26 @@ export interface Manifest {
   name: string;
   builds: Build[];
 }
+
+export interface FlashState {
```

Comment:
We should create a state object per state-string and then make `FlashState` a union of these. That way we can type `details`.